### ISSUE TITLE
[Export] Upgrade to Stop-motion-OBJ 2.1

### ISSUE THREAD

**Issue by yuanming-hu** [taichi-dev/taichi#1589]
**Abstract**
Thanks to the great work by @neverhood311, [Stop-motion-OBJ](https://github.com/neverhood311/Stop-motion-OBJ) has upgraded to v2.1.

**TODO**
 - The documentation may need an update to v2.1: https://taichi.readthedocs.io/en/latest/export_results.html#import-ply-files-into-houdini-and-blender: (welcome contribution)
> Blender requires an add-on called Stop-motion-OBJ to load the result sequences. A very detailed explanation video is provided by the author on how to install, enable, and use the add-on. Just keep in mind that significant changes in the workflow were made in their most recent test releases. To follow their instruction readme and video, please use v2.0.2.
 - @archibate could you help to export your cloth simulation `ply` files, so that @neverhood311 can feature your simulation on his project page?
 - (any other related tasks we can do to better guide users to the Stop-motion-OBJ plugin?)

**Comment by archibate** [comment]
Hi, thank for purposing this. But sadly I can't access the link https://www.youtube.com/watch?v=Ja7F7WHD39Q due to my 'poor network' condition, would you mind to copy that video to bilibili so people like me could access it? Thanks in advance.
Btw, I'm currently writing a 3D version of mpm and using the cool `PLYWriter`.

**Comment by yuanming-hu** [comment]
> Hi, thank for purposing this. But sadly I can't access the link https://www.youtube.com/watch?v=Ja7F7WHD39Q due to my 'poor network' condition, would you mind to copy that video to bilibili so people like me could access it? Thanks in advance.

Sure I can help with that. @neverhood311 Do you mind I make a copy of your tutorial video and upload it to Bilibili, a Chinese video sharing platform? People in China do not have easy access to Youtube, unfortunately...

**Comment by archibate** [comment]
Yes... BTW, I heard from #1638 that the video is serving for the old version, i.e. out-of-date. Do we have a video for latest version?

**Comment by neverhood311** [comment]
@yuanming-hu @archibate I made the video based on one of the earliest versions of the add-on, so it's not very useful for the latest version. I haven't had time to record an updated tutorial video, but I've created [pretty detailed documentation](https://github.com/neverhood311/stop-motion-obj/wiki) to help compensate for the absence of a new video. The documentation also includes a quick-start guide for two common use-cases as well as animated screen captures of those two cases.

That being said, I do eventually want to create an update video tutorial and I will upload a copy to Bilibili.
